README.md

```diff
@@ -327,6 +327,46 @@ index
 controller.abort()
 ```
 
+
+### Using Meilisearch behind a proxy <!-- omit in toc -->
+
+#### Custom request config <!-- omit in toc -->
+
+You can customize the request generated ex: add headers
+
+```ts
+const client: MeiliSearch = new MeiliSearch({
+    host: 'http://localhost:3000/api/meilisearch/proxy',
```

Comment:
```suggestion
    host: 'http://localhost:7070/api/meilisearch/proxy',
```

as per the port we used in the other examples

Reply:
usually the proxy wouldn't be on the same port that's the whole idea